Editor/Window/AwsUserProfilesPage.cs

```diff
@@ -69,7 +69,7 @@ public AwsUserProfilesPage(VisualElement container, StateManager stateManager)
             {
                 ShowProfileMenu(_bootstrapMenu);
             };
-            BootstrapSettings = BootstrapSettingsFactory.Create();
+            BootstrapSettings = BootstrapSettingsFactory.Create(_stateManager);
```

Comment:
This should be included in the separate bootstrap PR

Reply:
Moved to bootstrap changes PR